### PR TITLE
noissue: fix e2e tests

### PR DESCRIPTION
**Pull request by atrigueiro**
**What's in this PR?**
Just updating the id of the Create Project button because it changed.
Screenshot 2023-06-09 at 12 34 38 am

**Why**
Because it's failing our e2e tests.

### CODE REVIEW

File: test/e2e/utils/jira.ts [hunk 1]
```diff
@@ -68,24 +68,24 @@ export const jiraAppUninstall = async (page: Page): Promise<Page> => {
...
+	await page.click("[data-testid='global-pages.directories.projects-directory-v3.create-projects-button']");
 	await page.click("[data-testid='project-template-select-v2.ui.layout.category-overview.template-list-card.template-list-button']");
 	await page.click("[data-testid='project-template-select-v2.ui.layout.screens.template-overview.template-overview-card.use-template-button.button']");
 	await page.click("[data-testid='project-template-select-v2.ui.layout.screens.project-types.footer.select-project-button-team-managed']");
 	await page.fill("[id='project-create.create-form.name-field.input']", projectId);
 	await page.click("[data-testid='project-access.field.button-trigger']");
 	await page.locator("[data-testid='project-access.field.dropdown-menu--content'] [data-testid='project-access.field.option']").nth(2).click();
 	await page.fill("[id='project-create.create-form.advanced-dropdown.key-field.input']", projectId);
-	await page.click("[data-test-id='project-create.create-form.create-screen.submit-button'] button");*/
+	await page.click("[data-test-id='project-create.create-form.create-screen.submit-button'] button");
```
Comment (rachellerathbone) [inline test/e2e/utils/jira.ts:88]: Looking at the screenshot, isn't the test id: `global-pages.directories.projects-directory-v3.create-projects-button`?

Comment (atrigueiro) [reply test/e2e/utils/jira.ts:88]: Yup that's the change on line 80 and it's working now. 
Michel left two blocks of code, one for v2 and one for v3 (comment on line 79). I just uncommented the v3 and commented the v2
Tests are now failing on line 85 but the id looks the same in our Jira so I'm not too sure there.

Comment (rachellerathbone) [reply test/e2e/utils/jira.ts:88]: Gotcha. Do we know what element data-test-id='project-create.create-form.create-screen.submit-button is for?

Comment (atrigueiro) [reply test/e2e/utils/jira.ts:88]: You can follow the whole flow of creating a project in Jira and that's what those ids are. This PR is not ready to be merged though bc it's still failing to find one of the buttons. I'll convert it to draft.

Comment (krazziekay) [reply test/e2e/utils/jira.ts:88]: There's a new page when creating a project, which asks to connect to other tools and thats messing up the flow with the old one. That's why its not working.

**Review by bgvozdev — APPROVED** [review]

**Pull request merged**